src/content/en/2022/cms.md

```diff
@@ -1,22 +1,568 @@
+
+We believe that the CMS plays a key role in the success of our collective efforts to build a fast and resilient web. Understanding the current state, asking questions, and posing lines of inquiry for future work is our path to achieving this goal.
+
+As a team, we’ve approached this year’s data with curiosity, and we’ve combined that curiosity with personal expertise with several of the most popular CMSs. We recommend that you take the comparisons and our analysis in stride, considering the variability between CMSs, and the differing types of user content that are built on these platforms.
+
+## What is a CMS?
+
+The term Content Management System (CMS) refers to systems enabling individuals and organizations to create, manage, and publish content. A CMS for web content, specifically, is a system aimed at creating, managing, and publishing content to be experienced on the web.
+
+Each CMS implements a range of content management capabilities and the corresponding mechanisms for users to build websites around their content. CMSs also provide administrative capabilities to facilitate the addition and management of content as needed.
+
+CMSs differ widely in the approaches they offer for building sites; some provide ready-to-use templates which are supplemented with user content, and others require user involvement for designing and constructing the site structure.
+
+When we think about CMSs, we try to account for all the parts that form an ecosystem surrounding the CMS platform, including hosting providers, extension developers, development agencies, site builders, etc. Thus, when we talk about a CMS, we usually refer to both the platform itself and its surrounding ecosystem.
+
+Our dataset, based on <a hreflang="en" href="https://www.wappalyzer.com/technologies/cms">Wappalyzer’s definition</a> of a CMS, identified over 270 individual CMSs. Know a CMS that’s missing? <a hreflang="en" href="https://github.com/wappalyzer/wappalyzer/blob/7ac625c34432cb35d01abd683f88d3bfadca4cca/CONTRIBUTING.md">Contribute to Wappalyzer</a>.
+
+Some CMSs in the dataset are open source (e.g., WordPress and Joomla) and some of them are proprietary (e.g., Wix and Squarespace). Some CMSs can be used on “free” hosted or self-hosted plans, and there are also options for using these platforms on higher-tiered plans even at the enterprise level.
+
+The CMS space as a whole is a complex, federated universe of CMS ecosystems, separated and at the same time intertwined.
+
+## CMS adoption
+
+Our analysis throughout this work includes desktop and mobile websites. The vast majority of URLs we looked at are in both datasets, but some URLs are only accessed by desktop or mobile devices. This can cause divergences in the data, and we thus look at desktop and mobile results separately.
+
+{{ figure_markup(
+  image="cms-adoption.png",
+  caption="47% of mobile sites were flagged as CMS's in 2022.",
```

Comment:
For figure captions we just give the title, rather than any interpretation:

```suggestion
  caption="CMS adoption.",
```

Can you similarly update the other figure captions?